src/dotnet-scaffolding/dotnet-scaffold-aspnet/Program.cs

```diff
@@ -147,17 +155,17 @@ public static void Main(string[] args)
 
         var runner = builder.Build();
         runner.RunAsync(args).Wait();
-        //    config.AddCommand<AreaCommand>("area");
-
     }
 
     static void ConfigureServices(IServiceCollection services)
```

Comment:
We should probably consider having a separate AddSteps method (or something similar) specifically for adding all the steps to the collection that's separate from the ConfigureServices just to keep the semantic intent obvious.

Reply:
yup sounds good, adding a TODO there.